pkgs/servers/http/nginx/modules.nix

```
@@ -18,12 +19,26 @@ let
 in
 
 {
+  fastcgi-cache-purge = throw "module fastcgi-cache-purge renamed to cache-purge";
+  ngx_aws_auth = throw "module fastcgi-cache-purge renamed to aws-auth";
```

Nitpick: Just to make it a sentence:

```suggestion
  fastcgi-cache-purge = throw "fastcgi-cache-purge was renamed to cache-purge";
  ngx_aws_auth = throw "fastcgi-cache-purge was renamed to aws-auth";
```

---

Fixed.